platforms/maven/opencv/pom.xml

```diff
@@ -175,14 +176,14 @@
                         <configuration>
                             <rules>
                                 <requireEnvironmentVariable>
-                                    <level>WARN</level>
+                                    <level>ERROR</level>
                                     <variableName>ANT_HOME</variableName>
                                     <message>$ANT_HOME is not set. Build may fail.</message>
```

Comment:
I agree with you but this assumes that the user is building using the system packages (I am assuming you mean the repo provided ones). I for one do not use the system JDK packages and install the Oracle JDK manually. I also do not set the JAVA_HOME environment variable set, mainly because Debian alternatives handle determination of which binaries to execute and OpenCv is one of a very few instances I need to have it set. Without JAVA_HOME set I see this build error:

`[INFO] 
[INFO] --- cmake-maven-plugin:3.4.1-b2:compile (cmake-compile) @ opencv ---
[INFO] Configured to use native CMake
make: *** No rule to make target 'opencv_java'. Stop.
`

Which to someone with a similar installation as mine i.e. using Oracle JDK may not link to JAVA_HOME not being set.

Consequently I decided to change these two checks to Error in an attempt to provide clear failure reason.